src/MessageList.jsx

```diff
@@ -0,0 +1,92 @@
+// @flow
+
```

Comment:
This will surely be broken up into smaller components. You can do it in this PR or later.